packages/nvidia_nat_langchain/src/nat/plugins/langchain/llm.py

```
@@ -366,3 +368,296 @@ async def _agenerate(self,
+                        messages=chat_messages,
+                        max_tokens=self.max_new_tokens,
+                        temperature=self.temperature,
+                        top_p=self.top_p,
+                        seed=self.seed,
+                        stop=stop,
+                        stream=True,
+                        **kwargs,
+                    ):
+                        # Use captured loop instead of get_event_loop()
+                        asyncio.run_coroutine_threadsafe(queue.put(chunk), loop)
+                except Exception as err:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)